source/GameLoadingPanel.h

```diff
@@ -13,6 +13,8 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #ifndef GAME_LOADING_PANEL_H_
 #define GAME_LOADING_PANEL_H_
 
+#include "Conversation.h"
+#include "ConversationPanel.h"
```

Comment:
Don't include the whole file since it's not needed, instead include it in the cpp file and add `class Conversation;`  below.

And the conversation panel header isn't used in this file either, so include it in the cpp file instead.

Reply:
I don't understand this feedback.  Can you be more specific with a code example?

Reply:
1. Delete the two lines you added (the two includes)
2. Add a line `class Conversation;` in this file (there are two similar lines already, so add it there).
3. Add the two lines you deleted in 1) in GameLoadingPanel.cpp

Reply:
Done.